File: VerificacaoParImpar.java
import java.util.Scanner;

public class VerificacaoParImpar {
    public static void main(String[] args) {
        
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite o numero: ");
        int numero1 = scanner.nextInt();
        if (numero1 % 2 == 0){
            System.out.println("O numero e par");
        }
        else{
            System.out.println("O numero e impar");
        }

        scanner.close();

    }
}

File: verificacao_par_impar.c
#include <stdio.h>

int main(void){

    int numero1;
    printf("Digite o numero: ");
    scanf("%d", &numero1);
    if (numero1 % 2 == 0){
        printf("\nO numero e par.");
    }
    else{
        printf("\nO numero e impar.");
    }

}